csvs/card.csv

```diff
@@ -2369,12 +2369,12 @@ MON280	MON	Pound for Pound	3	3	4	2			C	Generic, Action, Attack	Dominate				When
+MON284	MON	Stony Woottonhog	1	2	6	2			C	Generic, Action, Attack					While Stony Woottonhog is defended by less than 2 non-equipment cards, it has +1{p}.	Grandad Warthog leads the way.	Generic Action – Attack	Jessada Sutthi																	S R – MON284 – F, S R – MON284 – U	https://storage.googleapis.com/fabmaster/media/images/MON284.width-450.png - MON284 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON284.width-450.png - MON284 – U			
+MON285	MON	Stony Woottonhog	2	2	5	2			C	Generic, Action, Attack					While Stony Woottonhog is defended by less than 2 non-equipment cards, it has +1{p}.	Grandad Warthog leads the way.	Generic Action – Attack	Jessada Sutthi																	S R – MON285 – F, S R – MON285 – U	https://storage.googleapis.com/fabmaster/media/images/MON285.width-450.png - MON285 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON285.width-450.png - MON285 – U			
+MON286	MON	Stony Woottonhog	3	2	4	2			C	Generic, Action, Attack					While Stony Woottonhog is defended by less than 2 non-equipment cards, it has +1{p}.	Grandad Warthog leads the way.	Generic Action – Attack	Jessada Sutthi																	S R – MON286 – F, S R – MON286 – U	https://storage.googleapis.com/fabmaster/media/images/MON286.width-450.png - MON286 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON286.width-450.png - MON286 – U			
+MON287	MON	Surging Militia	1	2	5	2			C	Generic, Action, Attack					Surging Militia has +1{p} for each non-equipment card defending it.	The villages of the golden fields are just as noble and brave as the Solanians who live within the city.	Generic Action – Attack	Kyxarie Peralta																	S R – MON287 – F, S R – MON287 – U	https://storage.googleapis.com/fabmaster/media/images/MON287.width-450.png - MON287 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON287.width-450.png - MON287 – U			
+MON288	MON	Surging Militia	2	2	4	2			C	Generic, Action, Attack					Surging Militia has +1{p} for each non-equipment card defending it.	The villages of the golden fields are just as noble and brave as the Solanians who live within the city.	Generic Action – Attack	Kyxarie Peralta																	S R – MON288 – F, S R – MON288 – U	https://storage.googleapis.com/fabmaster/media/images/MON288.width-450.png - MON288 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON288.width-450.png - MON288 – U			
+MON289	MON	Surging Militia	3	2	3	2			C	Generic, Action, Attack					Surging Militia has +1{p} for each non-equipment card defending it.	The villages of the golden fields are just as noble and brave as the Solanians who live within the city.	Generic Action – Attack	Kyxarie Peralta																	S R – MON289 – F, S R – MON289 – U	https://storage.googleapis.com/fabmaster/media/images/MON289.width-450.png - MON289 - F, https://storage.googleapis.com/fabmaster/media/images/U-MON289.width-450.png - MON289 – U			
```

Comment:
"villages" vs "villagers" was actually a typo in the first edition of Monarch that they fixed in Unlimited - "villagers" should be correct!